### DISCUSSION TITLE
where can I get the code of  bias detector example

### DISCUSSION THREAD

**Discussion by homily707** [kubeflow/kfserving#2520] [Q&A] [answered]
the bias detector example from [here](https://kserve.github.io/website/master/modelserving/detect/aif/germancredit/#process-payload-logs-for-metrics-calculation) just said run `python json_from_logs.py` , but where can I get the code of file json_from_logs.py

**Comment by jagadeeshi2i** [comment, marked as answer]
https://github.com/kserve/kserve/tree/master/docs/samples/explanation/aif/germancredit

**Reply by homily707** [reply]
thanks a lot
